src/App.js

```diff
@@ -2,19 +2,14 @@ import React from "react";
 import Home from "./components/Home";
 import { Link, Route } from "react-router-dom";
 import Form from "./components/Form";
+import Navbar from "./components/Navbar";
 
 const App = () => {
   return (
     <div>
-      <header>
-        <h1>Lambda Eats</h1>
-        <Link to="/">Home</Link>
-        <Link to="/pizza">
-          <button>Pizza?</button>
-        </Link>
-        <Route exact path="/" component={Home} />
-        <Route path="/pizza" component={Form} />
-      </header>
+      <Navbar />
+      <Route exact path="/" component={Home} />
+      <Route exact path="/pizza" component={Form} />
```

Comment:
React Router has some newer syntax which helps with passing props. It might be worth trying to work with this on your spare time. 
`<Route path='/'><Home></Route>`